src/render/js/components/GenerateUUID.js

```diff
@@ -0,0 +1,62 @@
+import m from 'mithril'
+import { ipcRenderer } from 'electron'
+
+import sendToChannel from '../../../lib/sendToChannel'
+
+import UuidItem from './UuidItem'
+
+const sendTask    = sendToChannel(ipcRenderer, 'task')
+const requestUUID = fn => prop => () => fn({ type: 'uuid', num: prop() })
```

Comment:
Should set this up like
`(fn, prop) => () => fn({ type: 'uuid', num: prop() })`